cogs/logging/logging.py

```diff
@@ -179,6 +179,28 @@ async def timezone_set(self, ctx, *, timezone: zoneinfo.ZoneInfo):
         await Timezones.insert(update_on_conflict=Timezones.timezone, user_id=ctx.author.id, timezone=timezone.key)
         await ctx.tick()
 
+    @timezone.command(name='get')
+    async def timezone_get(self, ctx, *, user: discord.Member):
+        """Get a member's timezone."""
```

Comment:
Default to ctx.author if no user is passed.

```suggestion
    async def timezone_get(self, ctx, *, user: discord.User = None):
        """Get a user's timezone.
        
        `user`: The user who's timezone to display, defaults to you.
        """
        user = user or ctx.author
```